packages/stream_chat_flutter/lib/src/message_widget.dart

```diff
@@ -146,104 +146,110 @@ class MessageWidget extends StatefulWidget {
+      this.allRead = false,
+      this.onQuotedMessageTap,
+      this.onAttachmentTap})
+      : attachmentBuilders = {
           'image': (context, message, attachment) {
             return ImageAttachment(
-              attachment: attachment,
-              message: message,
-              messageTheme: messageTheme,
-              size: Size(
-                MediaQuery.of(context).size.width * 0.8,
-                MediaQuery.of(context).size.height * 0.3,
-              ),
-              onShowMessage: onShowMessage,
-              onReturnAction: onReturnAction,
-            );
+                attachment: attachment,
+                message: message,
+                messageTheme: messageTheme,
+                size: Size(
+                  MediaQuery.of(context).size.width * 0.8,
+                  MediaQuery.of(context).size.height * 0.3,
+                ),
+                onShowMessage: onShowMessage,
+                onReturnAction: onReturnAction,
+                onAttachmentTap: () {
+                  onAttachmentTap(message, attachment);
+                });
           },
           'video': (context, message, attachment) {
             return VideoAttachment(
-              attachment: attachment,
-              messageTheme: messageTheme,
-              size: Size(
-                MediaQuery.of(context).size.width * 0.8,
-                MediaQuery.of(context).size.height * 0.3,
-              ),
-              message: message,
-              onShowMessage: onShowMessage,
-              onReturnAction: onReturnAction,
-            );
+                attachment: attachment,
+                messageTheme: messageTheme,
+                size: Size(
+                  MediaQuery.of(context).size.width * 0.8,
+                  MediaQuery.of(context).size.height * 0.3,
+                ),
+                message: message,
+                onShowMessage: onShowMessage,
+                onReturnAction: onReturnAction,
+                onAttachmentTap: () {
+                  onAttachmentTap(message, attachment);
+                });
           },
           'giphy': (context, message, attachment) {
             return GiphyAttachment(
-              attachment: attachment,
-              messageTheme: messageTheme,
-              message: message,
-              size: Size(
-                MediaQuery.of(context).size.width * 0.8,
-                MediaQuery.of(context).size.height * 0.3,
-              ),
-              onShowMessage: onShowMessage,
-              onReturnAction: onReturnAction,
-            );
+                attachment: attachment,
+                messageTheme: messageTheme,
+                message: message,
+                size: Size(
+                  MediaQuery.of(context).size.width * 0.8,
+                  MediaQuery.of(context).size.height * 0.3,
+                ),
+                onShowMessage: onShowMessage,
+                onReturnAction: onReturnAction);
```

Comment:
missing trailing comma
also, how come this has no onAttachmentTap param?

Reply:
i find no value to do it :)